server/stream.go

```diff
@@ -805,29 +805,31 @@ func (mset *stream) rebuildDedupe() {
 
 	mset.ddloaded = true
 
-	// We have some messages. Lookup starting sequence by duplicate time window.
-	sseq := mset.store.GetSeqFromTime(time.Now().Add(-mset.cfg.Duplicates))
-	if sseq == 0 {
-		return
-	}
+	if mset.cfg.Duplicates != time.Duration(0) {
```

Comment:
Would do `> 0`

Reply:
Can not be just `0` as it's a time.Duration, I changed to `> time.Duration(0)`